cluster/ci/config/prow/jobs/cluster-launch-e2e.yaml

```diff
@@ -176,12 +178,12 @@ objects:
         #!/bin/bash
         set -euo pipefail
 
-        trap 'rc=$?; if [[ $rc -ne 0 ]]; then touch /tmp/exit; fi; exit $rc' EXIT
```

Comment:
Please don't change just a single job.  You need to make the other containers consistent with this, otherwise this becomes a nightmare to debug.

Reply:
The other launch configurations.